source/_integrations/emulated_kasa.markdown

```diff
@@ -37,6 +37,11 @@ emulated_kasa:
     light.dining_room:
       power: 40.2
 ```
+<div class='note'>
+
+Home Assistant must be restarted for changes to take effect (reloading YAML isn't sufficient).
```

Comment:
This is not specific to this integration. Adding this note here is odd; mostly because this should be part of the general YAML configuration documentation.

Adding it just on one very specific place, and not all others, imho is not good.

../Frenck